FlightScheduler/UserToken.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UserToken.swift
+//  FlightScheduler
+//
+//  Created by macbook on 27/02/2020.
+//  Copyright © 2020 Practice. All rights reserved.
+//
+
+import Foundation
+
+class UserToken: Codable {
```

Comment:
This class is a little confusing. We do such a class to decode the result we get from the backend. I don't understand what this does.

Reply:
Changes made